packages/preview/tuhi-alumni-vuw/0.1.0/typst.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "tuhi-alumni-vuw"
+version = "0.1.0"
+entrypoint = "tuhi-alumni-vuw.typ"
+authors = ["baptiste <https://github.com/baptiste>"]
+license = "MPL-2.0"
+description = "A template for VUW assessments."
```

Comment:
I think this was copied from another of your packages, you probably want to update it.

Reply:
Oops, good catch!